packages/components/src/style-provider/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { CacheProvider } from '@emotion/core';
+import { CacheProvider } from '@emotion/react';
```

Comment:
Down there when we invoke `createCache`, we'll need to provide it with a `key` prop, otherwise I'm getting an error:

![](https://cldup.com/0_-_aoaliU.png)

Simply changing:

```
return createCache( { container } );
```

to something like:

```
return createCache( { key: 'something', container } );
```

should do the job IMHO.

Reply:
But `StyleProvider` could be used any number of places. I think we'd probably want an instance id?

Reply:
Yeah, but at the same time do we really need multiple caches?

Reply:
Yes 🙂 We need one per iframe for iframes to receive styling.